pkg/api/types.go

```diff
@@ -441,10 +441,10 @@ const (
 
 // Represents a Persistent Disk resource in Google Compute Engine.
 //
-// A GCE PD must exist and be formatted before mounting to a container.
-// The disk must also be in the same GCE project and zone as the kubelet.
-// A GCE PD can only be mounted as read/write once.
-// GCE PDs support ownership management and SELinux relabeling.
+// A GCE PD must exist before mounting to a container. The disk must
+// also be in the same GCE project and zone as the kubelet. A GCE PD
+// can only be mounted as read/write once. GCE PDs support ownership
```

Comment:
> `can only be mounted as read/write once`

While, we're cleaning up the comments, this doesn't seem right. GCE PDs can be mounted as read-only as well right? Ditto for AWS below?